# 领航无人机的参考坐标系的信息
float64 yaw

geometry_msgs/Vector3 position
geometry_msgs/Vector3 velocity
geometry_msgs/Vector3 acceleration